### PR TITLE
Tests: add some!

### PR DESCRIPTION
**Pull request by danielbeardsley**
Add the minimal mocha test framework and
full tests for _js()

Note: built on top of #1

### CODE REVIEW

**Comment by danielbeardsley** [comment]
deploy_block 👍  on #1 being merged

**Comment by jarstelfox** [comment]
CR 🌵

File: test/window.translations.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const translations = {};
+window = {
```
Comment (djmetzle) [inline test/window.translations.js:2]: Is window declared but empty?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+language: node_js
+node_js:
+    - 8
```
Comment (djmetzle) [inline .travis.yml]: 8 is EOL in about 3 months. Why not use current?

File: test/window.translations.js (hunk 1 above)
Comment (danielbeardsley) [reply test/window.translations.js:2]: No, it was an undefined var in the node context.

File: .travis.yml (hunk 2 above)
Comment (danielbeardsley) [reply .travis.yml]: Sure!

**Comment by danielbeardsley** [comment]
un_deploy_block 👍   #1 is merged

**Comment by danielbeardsley** [comment]
QA 👍 

image

File: test/window.translations.js (hunk 1 above)
Comment (djmetzle) [reply test/window.translations.js:2]: So you don't want to declare it?

Comment (danielbeardsley) [reply test/window.translations.js:2]: If we declare it, it won't work.

In the js module system, doing:
```
window = {}
```
Will make a global variable available to every other module.

While:
```
const window = {};
```
(or var or let) will make a local variable only available to the current module.

**Comment by danielbeardsley** [comment]
This adds tests on something our app depends on. Can I get a re-CR @jarstelfox ?

**Comment by jarstelfox** [comment]
CR 🌵

**Pull request merged**